src/elements/content-sidebar/additional-tabs/__tests__/AdditionalTabFtuxTooltip.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import AdditionalTabFtuxTooltip from '../AdditionalTabFtuxTooltip';
+import Tooltip from '../../../common/Tooltip';
+
+describe('elements/content-sidebar/additional-tabs/AdditionalTabFtuxTooltip', () => {
+    const getWrapper = (props, children) =>
+        shallow(<AdditionalTabFtuxTooltip {...props}>{children}</AdditionalTabFtuxTooltip>);
+
+    test('should render the FTUX tooltip when isFtuxVisible is true and the FTUX can be shown', () => {
+        const children = <div>Child content</div>;
+        const targetingApi = () => ({
+            canShow: true,
+        });
+        const text = 'FTUX Text';
+
+        const wrapper = getWrapper({ isFtuxVisible: true, ftuxTooltipData: { targetingApi, text } }, children);
+
+        const ftuxTooltip = wrapper.find('.bdl-AdditionalTabFtuxTooltip');
```

Comment:
Can we target this with a `data-testid` attribute or component name? Same for the `.find('div')` and other similar selectors below.